### PR TITLE
Set FieldManager in managedFields to track CVO activity

### PR DESCRIPTION
**Pull request by vrutkovs**
This should visualize which fields are managed by CVO, showing when was the object last updated and which fields were set by CVO. 
Hopefully this would help us find out which fields are extra and relieve some work from controller.

@sttts any other advice how to get more info on this?

### CODE REVIEW

File: lib/manifest.go [hunk 1]
```diff
@@ -116,3 +116,17 @@ func ParseManifests(r io.Reader) ([]Manifest, error) {
 		manifests = append(manifests, m)
 	}
 }
+
+// Metav1CreateOptions returns default create options for new objects
+func Metav1CreateOptions() metav1.CreateOptions {
+	return metav1.CreateOptions{
+		FieldManager: "cluster-version-operator",
```
Comment (sttts) [inline lib/manifest.go:123]: this default to the user-agent. Is the user-agent wrong in CVO ?

**Comment by sttts** [comment]
/hold

I think this the wrong approach. The field manager default to the user-agent. Is the user-agent not set correctly in CVO? If it isn't, let's fix it.

File: lib/manifest.go (hunk 1 above)
Comment (vrutkovs) [reply lib/manifest.go:123]: Ah, right, its being set, sorry

**Comment by vrutkovs** [comment]
This field is already being set, no need to rewrite it

**Pull request closed without merging**